pyxdf/pyxdf.py

```diff
@@ -328,7 +328,16 @@ def load_xdf(
             elif tag == 6:
                 # read [StreamFooter] chunk
                 xml_string = f.read(chunklen - 6)
-                streams[StreamId]["footer"] = _xml2dict(fromstring(xml_string))
+                try:
+                    streams[StreamId]["footer"] = _xml2dict(fromstring(xml_string))
+                except Exception as e:
+                    if type(e).__name__ == 'ParseError':
+                        logger.error(
+                            "found likely XDF file corruption (%s), "
+                            "ignoring corrupted XML element in footer." % e
+                        )
+                    else:
+                        raise
```

Comment:
Can you simplify this by catching only a `ParseError` exception in the first place? No need for the `if`/`else` clause then. Thanks!

Reply:
Okay. Thank you!

Reply:
One last nitpick, would you mind using an f-string?

Reply:
I am happy to, I just thought I should be consistent with the other error messages. Let me know if it looks good!